snap/pack/pack_test.go

```diff
@@ -346,7 +346,7 @@ func (s *packSuite) TestDebArchitecture(c *C) {
 	c.Check(pack.DebArchitecture(&snap.Info{Architectures: nil}), Equals, "all")
 }
 
-func (s *packSuite) TestPackSimple(c *C) {
```

Comment:
I think these names got swapped accidentally at some point, `TestPackSimple` was testing packing a component, and `TestPackComponentSimple` was testing packing a snap.